flipclock7.info

```diff
@@ -3,6 +3,7 @@ description = Provides a field displaying an animated countdown clock.
 core = 7.x
 php = 7.0
 stylesheets[all][] = css/flipclock7.css
+stylesheets[all][] = css/flipclock7_admin.css
```

Comment:
Perhaps it’s better to add this via `#attached` to the field widget — otherwise it will be loaded always, even for the front end.